src/components/settings/PushNotifications/index.tsx

```diff
@@ -140,7 +140,7 @@ export const PushNotifications = (): ReactElement => {
                         <FormControlLabel
                           control={<Switch checked={!!preferences} onChange={handleOnChange} />}
                           label={preferences ? 'On' : 'Off'}
-                          disabled={!isOk || isRegistering}
+                          disabled={!isOk || isRegistering || !safe.deployed}
```

Comment:
in SignMessage we do `const isDisabled= ` which I prefer, but ok :)

Reply:
Agree, with that many variables it makes sense to extract it into a new one for better readability.

Reply:
One issue is that here we have the `isOk` variable which comes from the HoC so we can't extract that easily.